matrixConsole/AppDelegate.m

```diff
@@ -362,6 +372,27 @@ - (void)applicationWillTerminate:(UIApplication *)application
     // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
 }
 
+#pragma mark - Handling force touch shortcuts
+
+- (void)application:(UIApplication *)application performActionForShortcutItem:(UIApplicationShortcutItem *)shortcutItem completionHandler:(void (^)(BOOL))completionHandler {
+    
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:[NSBundle mainBundle]];
+    
+    UITabBarController *tabBarVC = [storyboard instantiateViewControllerWithIdentifier:@"MasterTabBarController"];
```

Comment:
Could you please use here the "masterTabBarController" property defined in AppDelegate instance?
In that case no change is required in Main.storyboard file.